.azure_pipelines/olive-aml-ci.yaml

```diff
@@ -0,0 +1,64 @@
+trigger:
```

Comment:
This trigger should probably be set to none? Otherwise, it will trigger the pipeline everytime a pr merges

Reply:
I intentionally added this trigger. While this is not blocking the PR, it might be okay to run this when code got merged. But this just reminds me that I probably should remove scheduled trigger in this case.

Reply:
yes, if we want this trigger then the schedule trigger is unnecessary. Since these tests don't take too long, it sounds okay to run them post merge. 
could you add `batch: true` like in https://github.com/microsoft/Olive/blob/ff929df6a3bd5b3082641f5ce34b728ba2361e7a/.azure_pipelines/build-doc.yaml#L8? Just in case multiple PR merge together, there is no need to run once for each

Reply:
Sounds good. Updated.